facebook/OpenBIC
# fby3.5: bb: Fix IPMB TX/RX

fby3.5: bb: Fix IPMB TX/RX
Summary:
- Fix BB IPMB TX/RX threads can't malloc successfully problem.
- Move MAX_IPMB_IDX to avoid forgetting chagne its value when adding a new IPMB task.

Dependency: #236 

Test plan:
1. Check BB IPMB is normal.
- Get BB fw version
root@bmc-oob:~# fw-util slot1 --version bb_bic
BB Bridge-IC Version: oby35-bb-v2022.01.01

- Get BB device ID
root@bmc-oob:~# bic-util slot1 0xe0 0x2 0x9c 0x9c 0x0 0x10 0x18 0x01
9C 9C 00 10 07 01 00 00 80 01 01 02 BF 9C 9C 00
00 00 00 00 00 00

2.  Check CL IPMB is normal.
- Get CL fw version
root@bmc-oob:~# fw-util slot1 --version bic
SB Bridge-IC Version: oby35-cl-v2022.06.51

- Get CL device ID
root@bmc-oob:~# bic-util slot1 0x18 0x01
00 80 11 06 02 BF 9C 9C 00 00 00 00 00 00 00
